website/content/partials/known-issues/1_16-default-lcq-pre-1_9-upgrade.mdx

```diff
@@ -0,0 +1,48 @@
+### Default lease count quota enabled when upgrading from Vault versions before 1.9
+
+#### Affected versions
+
+- 1.16+
+
+#### Issue
+
+Default lease count quotas are enabled in v1.16+ for new installs by consulting
+the version store.
+
+Before version 1.9, Vault did not maintain version history. As a result,
+upgrading from a version prior to 1.9 automatically enables a default lease
+count quota.
+
+Before performing an upgrade from Vault versions prior to 1.9 to versions 1.16+,
+first check the current number of unexpired leases via the
+[`vault.expire.num_leases`](/vault/docs/internals/telemetry#token-identity-and-lease-metrics)
+metric.
```

Comment:
```suggestion
Before you upgrade Vault from a version prior to 1.9 to versions 1.16+,
you should check the current number of unexpired leases via the
[`vault.expire.num_leases`](/vault/docs/internals/telemetry#token-identity-and-lease-metrics)
metric.
```

Style correction: Write in active voice and address the reader directly when providing explicit instruction